lib/src/widget/card/app_card.dart

```diff
@@ -0,0 +1,67 @@
+import 'package:flutter/material.dart';
+import 'package:xayn_design/src/linden/xsizes.dart';
+import 'package:xayn_design/xayn_design.dart';
+
+class AppCardWidget extends StatelessWidget {
+  /// Enable border line around card
+  final bool isHighlighted;
+
+  final BorderRadiusGeometry? borderRadius;
+
+  /// If not provided, then by default will apply [XSizes.elevationHigh]
+  final double? elevation;
+
+  /// If not provided, then by default will apply:
+  /// [EdgeInsets.all(XSizes.unit2)]
+  final EdgeInsets? contentPadding;
+
+  final Widget child;
+
+  const AppCardWidget({
+    Key? key,
+    required this.child,
+    this.isHighlighted = false,
+    this.elevation,
+    this.contentPadding,
+    this.borderRadius,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final linden = UnterDenLinden.getLinden(context);
+
+    final borderSide = BorderSide(
+      style: BorderStyle.solid,
+      width: 2,
```

Comment:
💅 We could use a `const kDefaultBorderWidth` variable for this value :)

Reply:
Sure, will update it